### PR TITLE
🛂 (client): Auth/login handling

### PR DESCRIPTION
**Pull request by tkhwang**
## Description

* client JWT, auth and login handling
* Apollo reactive variable 사용에 대해서는 향후 좀더 논의 예정. 우선 사용법 공유를 위하여 반영함.

## Type of change

- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] ✨ New feature (non-breaking change which adds new functionality)
- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ♻️ Refactor (non-breaking change which refactors code)
- [ ] 🩹 Chore (non-breaking change which changes the small things)
- [ ] 📝 This change requires a documentation update

## How Has This Been Tested?

- [X] Tested in local

### CODE REVIEW

File: apps/client/lib/withApollo.tsx [hunk 1]
```diff
@@ -13,7 +15,24 @@ const withApollo = nextWithApollo(
...
+                read() {
+                  return globalIsLoggedIn();
+                },
+              },
+              token: {
+                read() {
+                  return globalToken();
+                },
+              },
+            },
```
Comment (tkhwang) [inline apps/client/lib/withApollo.tsx:32]: * 향후 apollo client cache 에 global state 사용하는 것 잘 써 먹어야 할 예정.

Comment (zlrlo) [reply apps/client/lib/withApollo.tsx:32]: Apollo reactive variable...이것은 저도 문서를 읽어봐야겠어요📚

**Review by zlrlo — APPROVED** [review]

**Comment by zlrlo** [comment]
Apollo reactive variable...이것은 저도 문서를 읽어봐야겠어요📚

**Comment by tkhwang** [comment]
https://www.apollographql.com/blog/apollo-client/caching/local-state-management-with-reactive-variables/

**Pull request merged**